torch/_inductor/metrics.py

```diff
@@ -1,8 +1,15 @@
+from __future__ import annotations
```

Comment:
This __future__ import annotatiosn should not be needed in conjuction with the circular import, can you remove and it make sure it still works?

Reply:
I've removed the import annotations